### PR TITLE
Spree 4.2

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -81,6 +81,7 @@ gem 'deface'
 gem 'spree', github: 'Drakkar-Software/spree', branch: 'spree4.2'
 gem 'spree_auth_devise', github: 'Drakkar-Software/spree_auth_devise', branch: 'spree4.2'
 gem 'spree_gateway', github: 'Drakkar-Software/spree_gateway', branch: 'spree4.2'
+gem 'spree_sitemap', github: 'spree-contrib/spree_sitemap'
```
Comment (Herklos) [inline Gemfile:84]: Good idea :+1:

File: config/sitemap.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
+if Spree::Store.default.url.include? "https://"
```
Comment (Herklos) [inline config/sitemap.rb:1]: :+1:

File: db/default/spree/stores.rb [hunk 3]
```diff
@@ -3,7 +3,7 @@
   Spree::Store.new do |s|
     s.code              = 'OctoBot Community'
     s.name              = 'OctoBot Community'
-    s.url               = ENV['HOST_DOMAIN'] || 'obmp2.herokuapp.com'
+    s.url               = ENV['HOST_DOMAIN'] || 'community.octobot.online'
```
Comment (Herklos) [inline db/default/spree/stores.rb:6]: :+1:

**Review by Herklos — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,33 @@
 # OctoBot-Community
+
+## Installation
```
Comment (Herklos) [inline README.md:3]: :+1:

File: Gemfile [hunk 5]
```diff
@@ -64,28 +68,35 @@ group :test do
...
+  gem 'spree_credits', github: 'Drakkar-Software/spree_credits', branch: 'master'
+  gem 'spree_backtesting', github: 'Drakkar-Software/spree_backtesting', branch: 'master'
+  gem 'spree_tentacles', github: 'Drakkar-Software/spree_tentacles', branch: 'spree4.2'
+end
+
 # database
 gem 'activerecord-nulldb-adapter'
 
+# nexus connection: also have it here for sidekiq to have it
+gem 'nexus_api'
```
Comment (Herklos) [inline Gemfile:98]: It's not installed with spree_tentacles ?

File: config/spree_storefront.yml [hunk 6]
```diff
@@ -1,23 +1,92 @@
...
+#        - :title: OctoBot Hosting
+#          :url: /t/cloud/hosting
+#        - :title: Backtesting
+#          :url: /t/cloud/backtesting
+
+### default spree storefront ###
 default:
   navigation:
-    - title: Tentacles Packages
+    - title: Women
```
Comment (Herklos) [inline config/spree_storefront.yml:28]: Why replacing the customized storefront ?

File: public/robots.txt [hunk 7]
```diff
@@ -10,3 +10,4 @@ Disallow: /password
 Disallow: /api_tokens
 Disallow: /cart_link
 Disallow: /account_link
+Sitemap: https://www.community.octobot.online/sitemap.xml.gz
```
Comment (Herklos) [inline public/robots.txt:13]:
```suggestion
Sitemap: https://community.octobot.online/sitemap.xml.gz
```

File: Gemfile (hunk 5 above)
Comment (GuillaumeDSM) [reply Gemfile:98]: it is but it's not available in sidekiq if we dont list it here (sidekiq is load loading engines)

File: config/spree_storefront.yml (hunk 6 above)
Comment (GuillaumeDSM) [reply config/spree_storefront.yml:28]: I'll add a WIP, it's to display something on the interface for the moment

File: Gemfile (hunk 5 above)
Comment (Herklos) [reply Gemfile:98]: ok

**Review by Herklos — APPROVED** [review]

**Pull request merged**